engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/error/ArithmeticError.java

```diff
@@ -7,6 +7,11 @@
 
 @BuiltinType
 public class ArithmeticError extends UniquelyConstructibleBuiltin {
+  @Override
+  protected String getConstructorName() {
+    return "Error";
+  }
```

Comment:
I'm wondering if it would be worth creating a subclass with that override - then we could just override `ErrorBuiltin` or something like that to get this implementation without repetition?

Not sure if worth it, but may simplify a little bit.